lib/booking.rb

```diff
@@ -0,0 +1,81 @@
+require 'pry'
+require 'time'
+require 'date'
+
+module Hotel
+  class BookingSystem
+    attr_reader :rooms, :reservations
```

Comment:
The name of this class does not match the name of its file. This is fine in regular Ruby, but will be trouble once we get into Rails. You should rename the file to the snake case version of the class name: `booking_system.rb`.